.github/ISSUE_TEMPLATE/session.yml

```
@@ -98,9 +98,9 @@ body:
   - type: textarea
     id: comments
     attributes:
-      label: Comments (Optional)
+      label: Instructions for Meeting Planners (Optional)
```

We don't capitalize other labels.

```suggestion
      label: Instructions for meeting planners (Optional)
```